components/Navigation/ItemMobileNavigation.tsx

```diff
@@ -1,51 +1,47 @@
 import Link from "next/link";
 import React, { useState } from "react";
 import { Typography, VariantsTypography } from "../../ui/typography";
-import { ILinkNavigation } from "./Navigation";
 import {
   HoverLinkSC,
   LinkSC,
   SubcategorydLinkSC,
   SubNavigationMobile,
 } from "./styles";
+import { ILink, ILinkSubcategories } from "./types";
 
-export interface ILinkSubcategories {
-  name: string;
-  link: string;
-}
+export const ItemMobileNavigation = ({ link }: ILink) => {
+  const [open, isOpen] = useState(false);
 
-export const ItemMobileNavigation = ({
-  subcategories,
-  title,
-  href,
-  iconClose,
-  iconOpen,
-}: ILinkNavigation) => {
-  const [subnav, setSubnav] = useState<boolean>(false);
+  const handleMouseEnter = () => {
+    isOpen(true);
+  };
+  const handleMouseLeave = () => {
+    isOpen(false);
+  };
 
-  const showSubNav = () => setSubnav(!subnav);
   return (
     <>
-      <LinkSC onMouseEnter={subcategories && showSubNav}>
-        <Link href={href}>
+      <LinkSC
+        onMouseLeave={handleMouseLeave}
```

Comment:
this function will call always when you will leave the mouse. First you can check is your navigation item contain the subCategories and if not, it not need to call this function at all